packages/studio/src/components/IFramePortal.tsx

```diff
@@ -1,27 +1,50 @@
 import { createPortal } from "react-dom";
-import { Dispatch, PropsWithChildren, SetStateAction, useEffect } from "react";
+import {
+  Dispatch,
+  PropsWithChildren,
+  SetStateAction,
+  useEffect,
+  useRef,
+} from "react";
+import useStudioStore from "../store/useStudioStore";
+import { EXPANDED_VIEWPORTS } from "./Viewport/defaults";
```

Comment:
I noticed that if you select a mobile viewport then resize the window, it leaks out into the prop editor panel if the window is too small.  Is this something we want to fix in this item?

![image](https://github.com/yext/studio/assets/59857107/3c9164f3-444d-4552-a3d8-15a6f70563e8)

Reply:
There's also interesting behavior where if the window is small, it seems like the viewport will grow if the window is made larger-- however if the window is large and is made smaller, the viewport won't shrink. Also, when the viewport resizes, it also appears that the contents of the viewport remain the same size, which I think is unexpected. I think that all these quirks are just caused by window resizing.